src/modelbench/benchmarks.py

```diff
@@ -100,14 +103,22 @@ class GeneralPurposeAiChatBenchmarkV1(BenchmarkDefinition):
     - Hazard score = Fraction unsafe
     """
 
-    def __init__(self, locale: Locale):
+    def __init__(self, locale: Locale, evaluator="default"):
         self.locale = locale
+        self.evaluator = evaluator
         super().__init__()
 
-    def _make_hazards(self) -> Sequence[HazardDefinition]:
-        return [SafeHazardV1(hazard_key, self.locale) for hazard_key in SafeHazardV1.all_hazard_keys]
+    def key(self):
```

Comment:
Are we certain that the benchmark should have identical TOML content for different evaluators? I think there should probably be differences in the UI that indicate which evaluator was used.

Reply:
There perhaps should be, but currently there isn't. There's already a bunch of duplication in the TOML and I didn't want to make that worse for no gain. Personally I'm hoping we throw out the TOML shortly, as it was based on a theory — non-engineers keeping the TOML updated — that didn't pan out.

Maybe add a card for that? A simple solution might be putting the UIDs into the info box at the bottom of the page.